pkg/result/ignore.go

```diff
@@ -83,19 +92,26 @@ func (i *IgnoreFinding) UnmarshalYAML(value *yaml.Node) error {
 
 type IgnoreFindings []IgnoreFinding
 
-func (f *IgnoreFindings) Match(id, path string, pkg *packageurl.PackageURL) *IgnoreFinding {
+func (f *IgnoreFindings) Match(id string, results *FindingsResults) *IgnoreFinding {
 	for _, finding := range *f {
 		if id != finding.ID {
 			continue
 		}
-		if !matchPath(path, finding.Paths) || !matchPURL(pkg, finding.PURLs) {
+		if !matchPath(results.TargetPath, finding.Paths) || !matchPURL(results.PackageURL, finding.PURLs) {
 			continue
 		}
 
-		log.Debug("Ignored", log.String("id", id), log.String("target", path))
-		return &finding
+		if finding.StartLine != 0 && finding.EndLine != 0 &&
+			(finding.StartLine != results.StartLine || finding.EndLine != results.EndLine) {
+			continue
+		}
 
+		log.Debug("Ignored", log.String("id", id), log.String("target", results.TargetPath),
+			log.String("startLine-endLine", fmt.Sprintf("%d-%d", results.StartLine, results.EndLine)))
```

Comment:
You can log the start and end separately

Reply:
Fair enough. I can make that adjustment.

Reply:
I guess to me this log line would change if the results are ignored in a range (start:end, would be one line like you currently have) but if not probably best to log separately.

Reply:
Updated!